### ISSUE TITLE
pip install fails

### ISSUE THREAD

**Issue by ejovi** [garbados/fullcontact.py#3]
With the following:

     File "<string>", line 20, in <module>
     File "/tmp/pip-build-ydnUXy/fullcontact.py/setup.py", line 19, in <module>
       long_description=file.read(open('README.md', 'r')),
   IOError: [Errno 2] No such file or directory: 'README.md'

**Comment by jgoldberg** [comment]
+1

**Comment by joestump** [comment]
Same. :+1:

**Comment by joestump** [comment]
After [reading over the docs](https://docs.python.org/2/distutils/sourcedist.html#specifying-the-files-to-distribute), it would appear that only `README` or `README.txt` (not `README.md`) are included by default. 

I suspect adding `README.md` to `data_files` in `setup.py` would fix this.

**Comment by cathalgarvey** [comment]
..also, file.read isn't valid Python3; I don't know how much more here is outdated but I'll issue a PR shortly with what I can find. :)

**Comment by garbados** [comment]
Updated on pypi. tho, if someone else wants to take over ownership of the project, please raise your hand.

**Comment by cathalgarvey** [comment]
Hmm, I'd willingly accept a role in maintaining this as a lightweight 
wrapper, and might even add a bit of feature-bloat, but I couldn't 
guarantee ongoing Py2 compatibility. I'd try, but it's always going to 
be secondary to me as I simply never use it! :)

Let me know if you'd like to unload and I'll oblige, though with all the 
inbound links to your Github it probably ought to remain your repo? Not 
sure how that works.

On 12/09/15 20:28, Diana Dinosaur wrote:
> Updated on pypi. tho, if someone else wants to take over ownership of
> the project, please raise your hand.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/garbados/fullcontact.py/issues/3#issuecomment-139811402>.
>

-- 
Scientific Director, IndieBio EU Programme
  Now running in Cork, Ireland May->July
  Learn more at indie.bio and follow along!
Twitter:  @onetruecathal
Phone: +353876363185
miniLock: JjmYYngs7akLZUjkvFkuYdsZ3PyPHSZRBKNm6qTYKZfAM
peerio.com: cathalgarvey

**Comment by garbados** [comment]
yeah, i can give you commit privileges to the repo, and to the pypi namespace. that'd help plenty.

**Comment by garbados** [comment]
done! thanks :)

**Comment by cathalgarvey** [comment]
Cool, thank you! Will try not to burn the place down. :)
